extensions/aegis/src/component/otp/OtpList.tsx

```diff
@@ -45,7 +67,7 @@ export function OtpList() {
           }
         />
       ) : (
-        <OtpListItems items={items.otpList} setItems={setItems} />
+        <OtpListItems items={filteredItems} setItems={setItems} />
```

Comment:
Did you try the [`keywords`](https://developers.raycast.com/api-reference/user-interface/list#props-2)?

You can add those fields to the `keywords` for searching.

Reply:
Changing `keywords` from `keywords={[subtitle]}` to `keywords={[subtitle, ...(item.issuer?.split(' ') || [])]}` allows matching sub-words of the issuer. However, I haven’t found a good way to match all possible substrings of the issuer. Do you have any ideas?

Reply:
I prefer to use the [fast-fuzzy](https://github.com/EthanRutherford/fast-fuzzy) for searching. It's very simple to use and powerful for searching.

```js
import {Searcher} from 'fast-fuzzy';

const searcher = new Searcher(items, { keySelector: (item) => [
	item.name,
	item.subtitle,
	item.foobar
] });

const filtered = searchText ? searcher.search(searchText) : items;
```